packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -24,7 +24,7 @@ export function buildPrivateNamesMap(props) {
         update.getId = prop.scope.generateUidIdentifier(`get_${name}`);
       } else if (prop.node.kind === "set") {
         update.setId = prop.scope.generateUidIdentifier(`set_${name}`);
-      } else if (prop.node.kind === "method" && isMethod && isInstance) {
+      } else if (prop.node.kind === "method" && isMethod) {
```

Comment:
Nit: I don't think that this `isMethod` check is needed, since we are already checking `kind`.